docker/Dockerfile-workers

```diff
@@ -1,12 +1,28 @@
 FROM centos:8 AS builder
 
+RUN dnf -y install dnf-plugins-core \
```

Comment:
Since this is CentOS, you can just install EPEL with `dnf install -y epel-release`.